go.mod

```diff
@@ -92,6 +92,7 @@ require (
 	github.com/spf13/cobra v1.8.0
 	github.com/stretchr/testify v1.9.0
 	github.com/thedevsaddam/retry v0.0.0-20200324223450-9769a859cc6d
+	github.com/tidwall/gjson v1.17.1
```

Comment:
In general I'm not against adding a new dependency if it helps solving a specific problem, what's the added value in this case for `gjson` lib? Wouldn't be possible to reuse something like the existing code, ie `nmstateconfig.getFirstIP`?

Reply:
We've switched to directly resolving the hostname. gjson lib has been removed.